javascript/javascript1/week1/goodboy-oldboy/goodboy-oldboy.js

```diff
@@ -0,0 +1,8 @@
+const dogYearOfBirth = 2012;
+const dogYearFuture = 2045;
+const dogYear = dogYearFuture - dogYearOfBirth;
```

Comment:
You are missing a boolean variable called `shouldShowResultInDogYears` that should change the logged message depending on true or false and switch between a message with dog years and human years. Please, have another read of the assignment.